app-admin/fleet/metadata.xml

```diff
@@ -3,7 +3,7 @@
 <pkgmetadata>
   <!-- maintainer-needed -->
   <longdescription lang="en">
-	</longdescription>
+    </longdescription>
```

Comment:
What's this? It doesn't seem relevant nor meaningful. I'm wondering how it got in there.